app/custom_domain_validation.py

```diff
@@ -0,0 +1,33 @@
+from app.db import Session
+from app.dns_utils import get_cname_record
+from app.models import CustomDomain
+
+
+class CustomDomainValidation:
+    def __init__(self, dkim_domain: str):
+        self.dkim_domain = dkim_domain
+
+    def get_dkim_records(self, custom_domain: CustomDomain) -> {str: str}:
+        """
+        Get a list of dkim records to set up. It will be
+
+        """
+        return {
+            (f"{key}._domainkey", f"{key}._domainkey.{self.dkim_domain}")
```

Comment:
I don't see `custom_domain` is used here, is this expected?

Reply:
Yes, The template gives only the record to the user, not the full domain. As in 
`Set dkim._domainkey record in your domain`

Reply:
it isn't used, should it be removed from the method params?

Reply:
Removed